components/BookList/BookList.js

```diff
@@ -30,21 +12,31 @@ export default function BookList({
     <StyledList>
       {books?.map((book) => (
         <li key={book.id}>
-          <StyledLink href={`/book-details/${book.id}`}>
-            <BookCard
-              title={book.title}
-              author={book.author}
-              genre={book.genre}
-              cover={book.cover}
-              booksInfo={booksInfo}
-              bookId={book.id}
-              handleToggleAlreadyRead={handleToggleAlreadyRead}
-              handleToggleBookmark={handleToggleBookmark}
-              handleToggleCurrentlyReading={handleToggleCurrentlyReading}
-            />
-          </StyledLink>
+          <BookCard
+            title={book.title}
+            author={book.author}
+            genre={book.genre}
+            cover={book.cover}
+            booksInfo={booksInfo}
+            bookId={book.id}
+            handleToggleAlreadyRead={handleToggleAlreadyRead}
+            handleToggleBookmark={handleToggleBookmark}
+            handleToggleCurrentlyReading={handleToggleCurrentlyReading}
+          />
         </li>
       ))}
     </StyledList>
   );
 }
+
+const StyledList = styled.ul`
+  list-style: none;
+  display: flex;
+  flex-direction: column;
+  gap: 1rem;
+  margin-left: auto;
+  margin-right: auto;
+  padding-left: 5px;
+  padding-right: 5px;
+  padding-bottom: 40px;
```

Comment:
You could work withe the shorthand properties.

```suggestion
  margin: 0 auto; 
  padding: 0 5px 40px; 
```